Source/Scene/Label.js

```diff
@@ -1030,6 +1044,42 @@ define([
                     }
                 }
             }
+        },
+
+        /**
+         * Determines whether or not run the reverseRtl algorithm on the text of the label
+         * @memberof Label.prototype
+         * @type {Boolean}
+         * @default false
+         *
+         * @example
+         * // Example 1.
+         * // Set a label's rtl during init
+         * var myLabelEntity = viewer.entities.add({
+         *   id: 'my label',
+         *   text: 'זה טקסט בעברית \n ועכשיו יורדים שורה',
+         *   rtl: true
+         * });
+         *
+         * @example
+         * // Example 2.
+         * var myLabelEntity = viewer.entities.add({
+         *   id: 'my label',
+         *   text: 'English text'
+         * });
+         * // Set a label's rtl after init
+         * myLabelEntity.rtl = true;
+         * myLabelEntity.text = 'טקסט חדש'
+         */
+        rtl : {
+            get : function() {
+                return this._rtl;
+            },
+            set : function(value) {
+                if (this._rtl !== value) {
+                    this._rtl = value;
```

Comment:
I think you should call `reverseRtl` here in addition to when setting the text.

Reply:
Thanks @hodbauer!

If I understand correctly, if `rtl` is set to false, and you update `text` to an rtl language, then set `rtl` to true, it won't display correctly:

```
label.text = 'שלום';
label.rtl = true;
```

That may be confusing to people using this feature.